app/models/medicaid_application_attributes.rb

```diff
@@ -102,4 +111,12 @@ def residential_or_homeless
       "Homeless"
     end
   end
+
+  def dependents
```

Comment:
Would it make sense to have tests around this logic (filtering dependents & the first name display) at this object level?

Reply:
done at 42daaa993bb04faaa6c973436262c8bee6aa135d